documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -139,6 +139,24 @@ you will need the new `hibernate-search-elasticsearch` jar.
 ----
 ====
 
+If your Elasticsearch instance is hosted on Amazon Web Services,
+and if your instance is not public,
+you will need an additional dependency to handle authentication:
+`hibernate-search-elasticsearch-aws`.
+
+.Maven dependencies for Hibernate Search with AWS-hosted Elasticsearch
```

Comment:
I don't think so.
First, because there is no such explanation anywhere else in the documentation, so if we start talking about Wildfly (and I'm not sure we should), it should be in a dedicated section.
Second, because if consume a Wildfly module, I'd be really surprised to have to include part of the library as modules, and part of it as JARs in my deployment... It would just be weird, wouldn't it?